services/licence-finder/docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '3.7'
+
+x-licence-finder: &licence-finder
+  build:
+    context: .
+    dockerfile: Dockerfile.govuk-base
+  image: licence-finder
+  volumes:
+    - ${GOVUK_ROOT_DIR:-~/govuk}:/govuk:delegated
+    - home:/home/build
+  working_dir: /govuk/licence-finder
+
+services:
+  licence-finder-lite: &licence-finder-lite
+    <<: *licence-finder
+    privileged: true
```

Comment:
Are we sure this is necessary?

Reply:
Tests couldn't run as Chrome was not available. If iirc this was required for chromedriver.

Reply:
Ah fair enough. I couldn't see where it was using it!